Dave-mash/StackOverflow-lite
# [finishes #162573571] Added a like icon

# What does this PR do?
* Creates an icon for liking an answer

# Description of Task to be completed?
The tasks for this PR is:
* Add an icon to represent a like button
# How should this be manually tested?
* Clone this repository on your machine:
$ git clone https://github.com/Dave-mash/stackOverflow-lite.git

* CD into the project folder on your machine
$ cd stackOverflow-lite

* Check out the branch for this PR
$ git branch ft-user-login-162573571

* Open the index.html file on your browser

Pivotal Tracker story associated with this PR
Story-ID #162573571

Screenshot:
![thumbs](https://user-images.githubusercontent.com/41325679/49901549-87086000-fe72-11e8-8686-9bc8801026f8.png)
